ugvc/mrd/featuremap_utils.py

```diff
@@ -254,6 +272,24 @@ def edit_vcf_header(self, header: pysam.VariantHeader) -> pysam.VariantHeader:
             f'Number=1,Type=String,Description="{self.motif_length_to_annotate}'
             'bases in the reference after variant">'
         )
+        header.add_line(
+            f"##INFO=<ID={self.NEXT_1}," f'Number=1,Type=String,Description="1 base in the reference after variant">'
+        )
```

Comment:
@doron-st taught me that it's better to use .add_meta (you can find his examples in the repo) than .add_line